packages/security/src/appSecurity/AppSecurity.ts

```diff
@@ -15,8 +15,21 @@ export class AppSecurity {
     } else {
       console.warn("Security configuration is missing. The Security module will not work properly.");
     }
+    console.info("Url: ", this.internalConfig.url);
+  }
+
+  // Get device info AEROGEAR-8773
+  public getClientData = async (): Promise<any> => {
+    const metricsBuilder: MetricsBuilder = new MetricsBuilder();
+    const metricsPayload: {
+      [key: string]: any;
+    } = {};
+    const metrics = metricsBuilder.buildDefaultMetrics();
+    for (const metric of metrics) {
```

Comment:
We are leaking this logic from metrics SDK.
We could add this as a separate method to the builder and simply reuse core method: https://github.com/aerogear/aerogear-js-sdk/blob/master/packages/core/src/metrics/MetricsService.ts#L101-L104

I think the requirement here is to build payload but not send it to the server so it makes sense (IMHO) to just extract and reuse the same function.

Reply:
While I agree that it would be better to have this functionality in metrics in the core. While we are developing on a feature branch it makes more sense to keep the code bases separate to avoid having to constantly rebase and fix conflicts on the feature branch. I will spin up a jira to move this functionally to the core once we have merged with master @wtrocki what do you think ?

Reply:
`code bases separate to avoid having to constantly rebase and fix conflicts on the feature branch` 

FYI There is no active development on js-sdk at the moment and there will be no active dev in the next 3 months. This functionality is just 3 lines that exist in the code. It is faster to move that than create jira.